test/check.js

```diff
@@ -7,5 +7,5 @@ var postcss = require('postcss');
 module.exports = function check( input, output, opts ) {
   var processor = postcss([lost(opts)]);
 
-  expect(processor.process(input).css).to.equal(output);
```

Comment:
...okay, now I might be confusing myself. You're right. 🤔

http://chaijs.com/guide/styles/#expect

...for some reason reading through the failing tests has me pondering what "expected" and "actual" mean.